configuration.nix

```diff
@@ -65,6 +65,7 @@ isMaximal: {
         enable = isMaximal;
         crates.enable = isMaximal;
       };
+      ruby.enable = isMaximal;
```

Comment:
I'm not sure if ruby is common enough to be enabled in the maximal mode. Set this to `false` by default, and move it to the section below.